spec/controllers/distributions_controller_spec.rb

```diff
@@ -30,17 +30,21 @@
     end
 
     describe "POST #create" do
-      it "redirects to #index on success" do
-        i = create(:storage_location)
-        p = create(:partner)
+      let!(:storage_location) { create(:storage_location) }
+      let!(:partner) { create(:partner) }
```

Comment:
You can reference a spec-global ivar: `@partner` if you just need a throwaway partner. This partner is scoped to the default organization.

Reply:
👍